templates/internet-of-things/secondary-nav.html

```diff
@@ -11,7 +11,10 @@ <h5 class="p-navigation--secondary__logo">
           <a class="breadcrumbs__link {% if request.path == '/internet-of-things' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things">概览</a>
         </li>
         <li class="breadcrumbs__item">
-          <a class="breadcrumbs__link {% if request.path == '/internet-of-things/core' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things/core">Core</a>
+          <a class="breadcrumbs__link {% if request.path == '/internet-of-things/core' %}p-link--active{% else %}p-link--soft{% endif %}" href="/internet-of-things/core">什么是UbuntuCore</a>
+        </li>
+        <li class="breadcrumbs__item">
+          <a class="breadcrumbs__link {% if request.path == '/internet-of-things/doc' %}p-link--active{% else %}p-link--soft{% endif %}" href="https://ubuntu.com/core/docs">Ubuntu Core文档</a>
```

Comment:
You don't need `{% if request.path == '/internet-of-things/doc' %}p-link--active{% else %}p-link--soft{% endif %}` here as it links to ubuntu.com

Reply:
Updated